bank2ynab/bank_handler.py

```diff
@@ -72,6 +73,7 @@ def run(self) -> None:
                 logging.info(
                     f"No output data from this file for this bank. ({e})"
                 )
+                logging.debug(traceback.format_exc())
```

Comment:
Without this, errors in plugins like the `I/O operation on closed file` bug are just silently swallowed